### PR TITLE
Allen added a Dockerfile and a Jenkinsfile

### PR DESCRIPTION
**Pull request by AllenG08**
Hello Rishabh,

I am Allen . I came across your profile on Quine and was impressed by the application you submitted for the SWOC. I took the liberty of enhancing it by adding a Dockerfile and a Jenkinsfile. I believe these additions will further improve the functionality of your app. I encourage you to take a look when you have a moment.

### CODE REVIEW

File: Dockerfile
Comment (geeekgod) [inline Dockerfile:1]: Hi @allengb208 
I think the following Docker file has development environment setup. I generally prefer using production environment setup for deployments and that's the recommended practice as well. 

Is it possible for you to refactor it and migrate it to production environment.

Also I would appreciate if you could also add `docker compose` file as well.

**Review by geeekgod — CHANGES_REQUESTED** [review]
@allengb208 

I would appreciate if you can update the `Dockerfile` and add a `docker compose` file as well.

File: Dockerfile
Comment (AllenG08) [reply Dockerfile:1]: Hi Rishabh , Ya sure if you wan't i can add some Kubernetes manifests too. That'd be better if you wan't to setup the app in production . However i'll add a docker-compose too.

**Review by geeekgod — APPROVED** [review]
LGTM, Merging!

File: Dockerfile
Comment (geeekgod) [reply Dockerfile:1]: Hi @allengb208 

Thanks for collaborating I'll be merging the changes, might add some of my own additions as well

**Pull request merged**